src/main/java/app/ErrorResponse.java

```diff
@@ -0,0 +1,37 @@
+package app;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@NoArgsConstructor
+@Getter
+public class ErrorResponse{
```

Comment:
this seems like a controller level response DTO, therefore should be in controller's package